src/components/Request/Pay/Pay.tsx

```diff
@@ -19,6 +20,20 @@ export const PayRequestLink = () => {
     const [estimatedGasCost, setEstimatedGasCost] = useState<number | undefined>(undefined)
     const [transactionHash, setTransactionHash] = useState<string>('')
     const [unsignedTx, setUnsignedTx] = useState<peanutInterfaces.IPeanutUnsignedTransaction | undefined>(undefined)
+    const { estimatePoints } = useClaimLink()
```

Comment:
`const { estimatePoints } = useClaimLink()`
Is this good code practice? UseClaimLink in request link?